src/app-feature/account-feature/pages/login-page/login-page.component.html

```diff
@@ -12,15 +12,15 @@ <h1 class="app-page__title">{{ getPageTitle() | translate }}</h1>
   <label class="app-form__element--block">
     <span class="app-form__element-label">{{ 'Email:' | translate }}</span>
     <span class="app-form__element-input">
-      <input type="text" class="app-form__text-input" formControlName="email" autofocus>
+      <input type="text" class="app-form__text-input" formControlName="email" name="email">
       <app-form-control-errors namespace="loginPage.errors" formControlPath="email"></app-form-control-errors>
```

Comment:
When the `autofocus` attribute is used on required fields, it triggers `ChangeDetection` in inside the `FormControlErrorsComponent` when the focused element is blurred during tests and this causes said e2e test to fail. This might be related to angular/protractor#4582

I'll keep an eye out for that story, for now I've removed the autofocus.